tests/unit/src/test/scala/tests/testProvider/TestSuitesFinderSuite.scala

```diff
@@ -0,0 +1,87 @@
+package tests
+
+import scala.concurrent.Future
+
+import scala.meta.internal.metals.MetalsEnrichments._
+
+import com.google.gson.Gson
+import com.google.gson.GsonBuilder
+
+class TestSuitesFinderSuite extends BaseLspSuite("testSuitesFinderSuite") {
+  val gson: Gson =
+    new GsonBuilder().setPrettyPrinting.disableHtmlEscaping().create()
+
+  test("munit-test-suite") {
+    for {
+      _ <- Future.successful(cleanWorkspace())
+      _ <- initialize(s"""|/metals.json
+                          |{
+                          |  "app": {
+                          |    "libraryDependencies" : [ "org.scalameta::munit:0.7.29", "org.scalatest::scalatest:3.2.1" ],
+                          |    "scalaVersion": "2.13.6"
+                          |  }
+                          |}
+                          |/app/src/main/scala/foo/bar/MyTestSuite.scala
+                          |package foo.bar
+                          |abstract class AbstractTestSuite1 extends munit.FunSuite 
```

Comment:
Could you change the test to contain different packages and nesting? I think it's fine to adjust this one. We should also add no package suite